pupil-spa/src/app/services/register-input/registerInput.service.spec.ts

```diff
@@ -68,7 +68,7 @@ describe('RegisterInputService', () => {
       // registeredInputService line 55, where the `idx` is set to 100 if the
```

Comment:
I believe these comments @ 67-70 are now not relevant anymore?

Reply:
Good spot 👍 
Now removed